### PR TITLE
AO3-6208 Remove the TagWrangling#discuss functionality

### PR DESCRIPTION
**Pull request by gmjimenezc**
# Pull Request Checklist

* [x] Have you read ["How to write the perfect pull request"](https://github.blog/2015-01-21-how-to-write-the-perfect-pull-request/)?
* [x] Have you read the [contributing guidelines](https://github.com/otwcode/otwarchive/blob/master/CONTRIBUTING.md)?
* [ ] Have you added [tests for any changed functionality](https://github.com/otwcode/otwarchive/wiki/Automated-Testing)?
* [x] Have you added the [Jira](https://otwarchive.atlassian.net) issue number
  as the *first* thing in your pull request title (e.g. `AO3-1234 Fix thing`)
* [x] Do you have fewer than 5 pull requests already open? If not, please wait
  until they are reviewed and merged before creating new pull requests.

## Issue

https://otwarchive.atlassian.net/browse/AO3-6208

## Purpose

What does this PR do?

* It removes the TagWrangling#discuss functionality (the function, tests, and all its related code).

## Testing Instructions

How can the Archive's QA team verify that this is working as you intended?

If you have a Jira account with access, please update or comment on the issue
with any new or missing testing instructions instead.

## References

Are there other relevant issues/pull requests/mailing list discussions?

## Credit

What name and pronouns should we use to credit you in the [Archive of Our Own's Release Notes](https://archiveofourown.org/admin_posts?tag=1)?

warlockmel (she/her)

### CODE REVIEW

File: app/views/tag_wranglings/discuss.html.erb [hunk 1]
```diff
@@ -1,41 +0,0 @@
...
-    	<td><%= link_to strip_tags(comment.comment_content[0..50]) + '...', comment, :title => strip_tags(comment.comment_content) %></td>
-    	<td><%= comment.count_all_comments %></td>
-    	<td>
-    	  <% if comment.pseud %>
-    	  <%= link_to comment.pseud.name, [comment.pseud.user, comment.pseud] %>
-    	  <% else %>
-    	  <%=h comment.name %>
-    	  <% end %>
-    	</td>
-    	<td><%= last_reply_by(comment) %></td>
```
[the remark this answers is not in the document]
Comment (gmjimenezc) [reply app/views/tag_wranglings/discuss.html.erb:32]: I already committed the deleted file. I deleted all the discuss.html.erb. Let me know if you need anything else.

[the remark this answers is not in the document]
Comment (redsummernight) [reply app/views/tag_wranglings/discuss.html.erb:32]: I mean the definition of that method, which is unused after you removed the view:

https://github.com/otwcode/otwarchive/blob/92d7fc675437c76b8f1e33751ac3957224b064f0/app/helpers/comments_helper.rb#L17-L26

[the remark this answers is not in the document]
Comment (gmjimenezc) [reply app/views/tag_wranglings/discuss.html.erb:32]: Oh ok, sorry. Making the changes...

[the remark this answers is not in the document]
Comment (redsummernight) [reply app/views/tag_wranglings/discuss.html.erb:32]: No worries!

**Review by redsummernight — APPROVED** [review]
Looks good to me, thank you!

**Pull request merged**